mythril/laser/ethereum/state/machine_state.py

```diff
@@ -110,6 +111,7 @@ def __init__(
         """
         self._pc = pc
         self.stack = MachineStack(stack)
+        self.rstack = MachineStack(rstack)
```

Comment:
What do you  think about using a more explicit name? `rstack` might not be super clear for new users/ us in a few months
```suggestion
        self.subroutine_stack = MachineStack(subroutine_stack)
```